Examples/UIExplorer/js/CameraRollView.js

```diff
@@ -92,11 +92,12 @@ var CameraRollView = React.createClass({
       batchSize: 5,
       imagesPerRow: 1,
       assetType: 'Photos',
-      renderImage: function(asset) {
+      renderImage: function(asset: Image) {
```

Comment:
This isn't my design but that's what they are - the `CameraRollView` is a `ListView` where each row's data is an array of `Image`s (as in instances of the `Image` RN component). See for example [`getInitialState`](https://github.com/facebook/react-native/blob/44317996f361db4347c626be1e1499acba58c281/Examples/UIExplorer/js/CameraRollView.js#L113) and [`_rowHasChanged`](https://github.com/facebook/react-native/blob/44317996f361db4347c626be1e1499acba58c281/Examples/UIExplorer/js/CameraRollView.js#L209). Apparently that's just used as a container for data from which the rendered `Image` is built.